atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolTest.java

```diff
@@ -97,7 +98,7 @@ public void setup() {
         when(config.poolRefreshIntervalSeconds()).thenReturn(POOL_REFRESH_INTERVAL_SECONDS);
         when(config.timeBetweenConnectionEvictionRunsSeconds()).thenReturn(TIME_BETWEEN_EVICTION_RUNS_SECONDS);
         when(config.unresponsiveHostBackoffTimeSeconds()).thenReturn(UNRESPONSIVE_HOST_BACKOFF_SECONDS);
-
+        when(config.credentials()).thenReturn(mock(CassandraCredentialsConfig.class));
```

Comment:
This class doesn't seem to like the `@Mock` attribute, but that might just be legacy. Happy to move all these `mock(..)` calls to `@Mock`, but went with this here for consistency.

Reply:
If you want to use `@Mock` on fields, you need to annotate Junit4 test with `@RunWith(MockitoJUnitRunner.class)` or Junit5 test with `@ExtendWith(MockitoExtension.class)`